src/WebJobs.Script.Grpc/Channel/GrpcWorkerChannel.cs

```diff
@@ -355,7 +355,7 @@ internal FunctionLoadRequest GetFunctionLoadRequest(FunctionMetadata metadata, M
                     Directory = metadata.FunctionDirectory ?? string.Empty,
                     EntryPoint = metadata.EntryPoint ?? string.Empty,
                     ScriptFile = metadata.ScriptFile ?? string.Empty,
-                    IsProxy = metadata.IsProxy()
+                    IsProxy = false
```

Comment:
Just curious, why do we need to keep `IsProxy` around if we're getting rid of it? Is there an instance where this could be true?

Reply:
I think we need to keep this b/c it's part of the protobuf contract (which we share between all versions of the host). I didn't want to pull it out at the risk of breaking workers.

Reply:
That makes a lot of sense, thanks :)